### PR TITLE
Added logging for PythonInterpreter.initialize, fixed bgcolor and stop button color in present mode

### PR DESCRIPTION
**Pull request by mpsftw**
I noticed the background color wasn't working in present mode on my last fix. While investigating I saw the same problem replicated in the other export classes. I switched everything to use the PApplet constants for compatibility and maintenance. I also added bgcolor and stopbutton color args to the PdeSketch to handle background color and stop button color.

### CODE REVIEW

File: runtime/src/jycessing/Runner.java [hunk 1]
```diff
@@ -321,6 +321,7 @@ public static synchronized void runSketchBlocking(
     props.setProperty("python.main.root", sketchDirPath);
 
     final String[] args = sketch.getPAppletArguments();
+	log("PythonInterpreter.initialize args: " + String.join(" ", args));
```
Comment (jdf) [inline runtime/src/jycessing/Runner.java:324]: nit: whitespace

File: runtime/src/jycessing/mode/PyEditor.java [hunk 2]
```diff
@@ -341,7 +342,8 @@ private void runSketch(final DisplayType displayType) {
 
     try {
       sketchService.runSketch(
-          new PdeSketch(sketch, sketchPath, displayType, location, locationType));
+				new PdeSketch(sketch, sketchPath, displayType, location, locationType,
```
Comment (jdf) [inline runtime/src/jycessing/mode/PyEditor.java:345]: Same. I'm guessing you're using tabs, maybe? I'm using spaces.

File: runtime/src/jycessing/mode/run/PdeSketch.java [hunk 3]
```diff
@@ -103,7 +107,9 @@ public File getHomeDirectory() {
         }
         break;
       case PRESENTATION:
-        args.add("--present");
+			args.add(PApplet.ARGS_PRESENT);
```
Comment (jdf) [inline runtime/src/jycessing/mode/run/PdeSketch.java:110]: Same.

**Comment by mpsftw** [comment]
Sorry about the whitespace problems! I was having issues with the formatter in eclipse. I thought I caught all of them. Do you want me to fix it and do another pull request?

**Pull request merged**